roles/executionenvironment/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Read Secret Configuration from connection_secret
+  kubernetes.core.k8s_info:
+    kind: Secret
+    api_version: v1
+    name: "{{ connection_secret }}"
+    namespace: "{{ ansible_operator_meta.namespace }}"
+  register: tower_connection_secret
+  when: connection_secret is defined
+
+- name: DEPRECATED Read Secret Configuration from tower_auth_secret
+  kubernetes.core.k8s_info:
+    kind: Secret
+    api_version: v1
+    name: "{{ tower_auth_secret }}"
+    namespace: "{{ ansible_operator_meta.namespace }}"
+  register: tower_connection_deprecated_secret
+  when: tower_auth_secret is defined
```

Comment:
This is only relevent for CRD that had the variable of tower_auth_secret (which was depricated and renamed to connection_secret). This is why we can remove this task.